bin/i18n/sync-codeorg-in.rb

```diff
@@ -44,43 +44,56 @@ def localize_level_content
   level_failure_message_overrides = Hash.new
   level_authored_hints = Hash.new
   level_callouts = Hash.new
+  level_block_categories = Hash.new
 
   Dir.glob("dashboard/config/scripts/levels/*.level").sort.each do |file|
     level_name = File.basename(file, ".*")
     File.open(file) do |data|
       level_xml = Nokogiri::XML(data, &:noblanks)
+
+      # Properties
       config = JSON.parse(level_xml.xpath('//../config').first.text)
 
-      # Instructions
+      ## Instructions
```

Comment:
Intentional?

Reply:
Yeah, I was intending this to be a "sub-heading" under the "Properties" heading, with "Blocks" as a new second top-level heading.

If that's unclear or confusing, though, I can undo it

Reply:
It's just a Ruby comment, yeah?  No strong preference — my initial reaction was that I don't know what it is 😛.